README.md

```diff
@@ -328,18 +345,15 @@ best-efforts basis with the following warnings:
 * Some security patches and features cannot be backported.
 * Dependencies cannot be kept up-to-date.
 
-## Versioning
-
-This library follows [Semantic Versioning](http://semver.org/).
-
-This library is considered to be at **preview**, ready for testing by customers.
+### Release cadence
 
-More Information:
-[Google Cloud Platform Launch Stages](https://cloud.google.com/terms/launch-stages)
+This project aims for a release on at least a monthly basis. If no new features
+or fixes have been added, a new PATCH version with the latest dependencies is
+released.
 
 ## Contributing
```

Comment:
h3?

Reply:
I believe it should be h2 even though this one is an h3 in the Python Connector, "Contributing to the project" shouldn't be nested to "Supported runtime versions" imo

Reply:
correct, supported runtimes should be h3 as well.